packages/client-twitter/src/environment.ts

```diff
@@ -29,8 +29,14 @@ const twitterUsernameSchema = z
 export const twitterEnvSchema = z.object({
     TWITTER_DRY_RUN: z.boolean(),
     TWITTER_USERNAME: z.string().min(1, "X/Twitter username is required"),
-    TWITTER_PASSWORD: z.string().min(1, "X/Twitter password is required"),
-    TWITTER_EMAIL: z.string().email("Valid X/Twitter email is required"),
+    // OAuth 1.0a credentials (required for Twitter API v2)
+    TWITTER_API_KEY: z.string().optional(),
+    TWITTER_API_SECRET: z.string().optional(),
+    TWITTER_ACCESS_TOKEN: z.string().optional(),
+    TWITTER_ACCESS_SECRET: z.string().optional(),
```

Comment:
It might be good to add better zod validation than just option strings, would improve debugging/logs for wrong values